Tests/test_file_png.py

```diff
@@ -786,18 +786,20 @@ class TestTruncatedPngPLeaks(PillowLeakTestCase):
     iterations = 100  # Leak is 56k/iteration, this will leak 5.6megs
 
     def test_leak_load(self):
```

Comment:
Is there any particular reason you have chosen to add your test case to `TestTruncatedPngPLeaks`, rather than `TestFilePng`? `TestTruncatedPngPLeaks` was originally added in #2541, which was partly about memory leaks, but your description so far doesn't mention memory at all.